contracts/strategy/InvestorStrategy.sol

```diff
@@ -2,37 +2,83 @@
 // Copyright Fathom 2023
 pragma solidity 0.8.19;
 
-import "./BaseStrategy.sol";
+import "./TokenizedStrategy.sol";
 import "./interfaces/IInvestor.sol";
 import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
 // solhint-disable
-contract InvestorStrategy is BaseStrategy {
+contract InvestorStrategy is TokenizedStrategy {
```

Comment:
?

Reply:
I had undo that change and adjusted code accordingly to previous version